### PR TITLE
Enhancement/42 tree select whole row clickable

### LINKED ISSUE #42: Selecting a term in vocabulary detail should be possible upon clicking the whole row
In term tree in vocabulary detail, clicking on a row does not redirect on the term detail, unless the click happens on the term label itself.

Make the whole row clickable.

### PR DESCRIPTION
**Pull request by ledsoft**
Resolves #42 .

### CODE REVIEW

File: src/component/term/TermMetadataEdit.tsx [hunk 1]
```diff
@@ -205,6 +214,19 @@ export class TermMetadataEdit extends React.Component<TermMetadataEditProps, Ter
...
                             </Col>
                         </Row>
+                        <Row>
+                            <Col xs={12}>
+                                <FormGroup>
+                                    <Label id="term-metadata-edit-status" className="attribute-label">
+                                        {i18n("term.metadata.status")}
+                                    </Label>
+                                    <br/>
+                                    <DraftToggle draft={this.state.draft === undefined || this.state.draft}
```
Comment (psiotwo) [inline src/component/term/TermMetadataEdit.tsx]: This logic (draft === undefined) implies draft true is replicated on several places. Consider extracting it into some method.

File: src/i18n/en.ts [hunk 2]
```diff
@@ -244,6 +244,7 @@ export default {
             "Abbreviations are not allowed ('Value added tax' instead of 'VAT'). The label is written in sentence case " +
             "- first letter in upper case, the others in lower case. Label should not be changed, as its change might impact " +
             "meaning of data described by this concept.",
+        "term.label.confirmed.disabled": "The status of the term is CONFIRMED, so its label should not change.",
```
Comment (psiotwo) [inline src/i18n/en.ts]: Why is CONFIRMED in upper case (cf. the Czech variant)

**Review by psiotwo — DISMISSED** [review]
lgtm, see comments

File: src/component/term/TermMetadataEdit.tsx (hunk 1 above)
Comment (ledsoft) [reply src/component/term/TermMetadataEdit.tsx]: This belongs to #45 .

File: src/i18n/en.ts (hunk 2 above)
Comment (ledsoft) [reply src/i18n/en.ts]: This belongs to #45 .

**Review by psiotwo — APPROVED** [review]
lgtm

**Pull request merged**